fuse.py

```diff
@@ -376,18 +382,32 @@ class fuse_operations(Structure):
         ('flag_reserved', c_uint, 29),
     ]
 
+# When importing "fuse", you can check for the availability of features with
+# 	__features__.get('feature_name', False)
+__features__ = {
+	'nanosecond_int': True,
+	'utime_omit_none': True,
+	'utime_now_auto': True,
+	'libfuse2': True,	# Unused, demonstrating future use of __features__ dict
+	'libfuse3': False	# Unused, demonstrating future use of __features__ dict
+	}
 
-def time_of_timespec(ts):
-    return ts.tv_sec + ts.tv_nsec / 10 ** 9
+UTIME_OMIT = (1 << 30) - 2
+UTIME_NOW = (1 << 30) - 1
 
-def set_st_attrs(st, attrs):
+
+def set_st_attrs(st, attrs, nanosecond_int=False):
```

Comment:
This is probably the main change that belongs in the `fuse.py` and we probably should change the interface to use nanoseconds instead of seconds like time_ns will provide in Python 3.7

Reply:
@terencehonles Read my pull request carefully, it already makes use of `time_ns` from Python 3.7 if available.

Reply:
I think you misunderstood my comment. I was saying the future interface *should* be an integer rather than a float like it is now. That was probably to make it work with `time.time()` as you have above, but it dropped the resolution of the `timespec` object and that should never have happened. I'm not sure if the original other realized the implications of using a float rather than an integer, and I would like to break that interface as soon as possible.